thirdeye/thirdeye-frontend/app/utils/anomalies-tree-parser.js

```diff
@@ -0,0 +1,563 @@
+import { isEmpty } from "@ember/utils";
+import { set } from "@ember/object";
+import moment from "moment";
+
+const CLASSIFICATIONS = {
+  METRICS: {
+    KEY: "metrics",
```

Comment:
please keep code style consistency. I remember we enable the single quote rule. You could run prettier to format it.

Reply:
Can you tell me what is the exact command for it? Is it tied to your code changes in https://github.com/apache/incubator-pinot/pull/6245? If so, I may not be able to do as your changes are not merged yet.

Reply:
I haven't realized my code hasn't merged yet. The command `npm run eslint-app` only can detect errors, but not fix them. You need to fix manually or set your vscode prettier with the rule of a single quote.